ui/js/directives/treeherder/resultsets.js

```diff
@@ -39,17 +39,11 @@ treeherder.directive('thAuthor', function () {
     return {
         restrict: "E",
         link: function (scope, element, attrs) {
-            var userTokens = attrs.author.split(/[<>]+/);
-            var email = "";
-            if (userTokens.length > 1) {
-                email = userTokens[1];
-            }
-            scope.authorName = userTokens[0].trim();
-            scope.authorEmail = email;
+            scope.authorEmail = attrs.author.split(/[<>]+/)[0].trim();
```

Comment:
> the push titlebar from showing author name (in most cases, this is actually the email address) to always showing the email address

I don't think this is true.

`attrs.author.split(/[<>]+/)[0].trim()` will give the name if author if `foo <bar@baz.com>`.

It may now be that the author is never in that form, but if so, why do the split instead of a replace (eg for `author='<bar@baz.com>'` or nothing at all (for eg `author='bar@baz.com'`)?